components-styled/StyledShowMore.jsx

```diff
@@ -10,6 +10,8 @@ const StyledShowMore = styled.div`
   align-items: center;
   color: var(--color);
 
+  //TODO pass hue instead of all those classes
```

Comment:
you can remove this line 🕵️

Reply:
Morten had told me that he often saw this in check24. The file is one that could make cosmetically better and I wanted to mark it. If you press shift + command + f you find the place then immediately.( is also only one)